src/components/profile-item/complete-profile.constants.js

```diff
@@ -0,0 +1,33 @@
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+import InfoIcon from '@mui/icons-material/Info'
+import SellIcon from '@mui/icons-material/Sell'
+import LeakAddIcon from '@mui/icons-material/LeakAdd'
+
+export const profileItems = [
+  {
+    name: 'photo',
```

Comment:
name move to variables.

title subtitle add i18n

Reply:
Fixed